internal/logging/sentry_replacer_test.go

```diff
@@ -57,50 +58,50 @@ func TestUUIDSplit(t *testing.T) {
 
 func TestARN(t *testing.T) {
 	buf := bytes.NewBufferString("")
-	repl := NewSentryReplacer(buf)
+	repl := NewSentryReplacer(zerolog.LevelWriterAdapter{Writer: buf})
 	_, _ = repl.Write([]byte("arn:aws:iam::4328974392798432:role/my-role-123\n"))
 	require.Equal(t, "?\n", buf.String())
 }
 
 func TestARNSplit(t *testing.T) {
 	buf := bytes.NewBufferString("")
-	repl := NewSentryReplacer(buf)
+	repl := NewSentryReplacer(zerolog.LevelWriterAdapter{Writer: buf})
 	_, _ = repl.Write([]byte("arn:aws:iam::"))
 	_, _ = repl.Write([]byte("4328974392798432:role/my-role-123\n"))
 	require.Equal(t, "?\n", buf.String())
 }
 
 func TestIPv4(t *testing.T) {
 	buf := bytes.NewBufferString("")
-	repl := NewSentryReplacer(buf)
+	repl := NewSentryReplacer(zerolog.LevelWriterAdapter{Writer: buf})
 	_, _ = repl.Write([]byte("read tcp 10.128.24.14:42094->10.0.217.126:6379: i/o timeout\n"))
 	require.Equal(t, "read tcp ?->?: i/o timeout\n", buf.String())
 }
 
 func TestFingerprint(t *testing.T) {
 	buf := bytes.NewBufferString("")
-	repl := NewSentryReplacer(buf)
+	repl := NewSentryReplacer(zerolog.LevelWriterAdapter{Writer: buf})
 	_, _ = repl.Write([]byte("pubkey with fingerprint (57:d4:13:ff:c0:74:51:50:41:ec:e1:cd:f1:88:b0:61)\n"))
 	require.Equal(t, "pubkey with fingerprint (?)\n", buf.String())
 }
 
 func TestAWSResourceID(t *testing.T) {
 	buf := bytes.NewBufferString("")
-	repl := NewSentryReplacer(buf)
+	repl := NewSentryReplacer(zerolog.LevelWriterAdapter{Writer: buf})
 	_, _ = repl.Write([]byte("instance ID 'i-0fe8a8adc1403f5b1' does not exist\n\n\n"))
 	require.Equal(t, "instance ID '?' does not exist\n\n\n", buf.String())
 }
 
 func TestGoogleProject(t *testing.T) {
 	buf := bytes.NewBufferString("")
-	repl := NewSentryReplacer(buf)
+	repl := NewSentryReplacer(zerolog.LevelWriterAdapter{Writer: buf})
 	_, _ = repl.Write([]byte("The resource 'projects/xxx' was not found\n"))
 	require.Equal(t, "The resource ? was not found\n", buf.String())
 }
 
 func TestAzureTime(t *testing.T) {
 	buf := bytes.NewBufferString("")
-	repl := NewSentryReplacer(buf)
+	repl := NewSentryReplacer(zerolog.LevelWriterAdapter{Writer: buf})
 	_, _ = repl.Write([]byte("'start time': '2023-06-24T19:34:34.2581206+00:00'\n"))
 	require.Equal(t, "'start time': '?'\n", buf.String())
 }
```

Comment:
There are zero tests for the newly introduced `WriteLevel` function, just add an extra `require` to each test.

Reply:
Will do :+1: